USAGE.md

```diff
@@ -1130,6 +1130,76 @@ Date.today.weekend? # => true
 Date.today.in_dayset?(:school) # => false
 ```
 
+### Rules, Scripts, and Scenes
+
+Execute a Rule, a Script or a Scene by UID
+
+```ruby
+rules[rule_uid].trigger
+```
+
+Use Scenes in combination with Semantic Model
+
+```ruby
+rule "adjust room scene" do
+  description "Trigger the appropriate scenes for the room on motion detection"
+  changed Motion_Sensors.members
+  run do |event|
+    # the Active_Scene item can be set to the Scene tag 
+    # that we wish to activate when motion is detected
+    # e.g. ACTIVE/RELAXING/MOVIE/READING
+    scene_to_trigger = event.state.on? ? Active_Scene.state.to_s : "OFF"
+
+    # Get the scenes for the room where motion was detected
+    scenes = rules.to_a
```

Comment:
yes, we should include Enumerable into Rules::Registry so that `tagged` can work directly